src/scripts/main.js

```diff
@@ -1,3 +1,24 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+
+const rows = Array.from(table.rows);
+
+rows.forEach(row => {
+  if (row.cells.length > 1) {
+    const secondCell = row.cells[1];
+    if (secondCell) {
+      const clonedCell = secondCell.cloneNode(true);
+      if (row.cells.length > 2) {
+        const lastCell = row.cells[row.cells.length - 1];
+        if (lastCell) {
+          row.insertBefore(clonedCell, lastCell);
+        } else {
+          row.appendChild(clonedCell);
```

Comment:
The `row.insertBefore(clonedCell, lastCell)` method will insert the cloned cell before the last cell. But in the `else` block, you are appending the cloned cell as the last cell. This is a bit confusing because it means that the cloned cell will always be the last cell, regardless of whether it is inserted before the last cell or appended as the last cell. It would be more clear if you always insert the cloned cell before the last cell, and only append it as the last cell if there are less than 3 cells in the row.